fern/apis/server/definition/ingestion.yml

```diff
@@ -185,9 +185,32 @@ types:
       id: optional<string>
       traceId: string
       name: string
-      value: double
+      value:
+        type: commons.CreateScoreValue
+        docs: The value of the score. Must be passed as string for categorical scores, and numeric for boolean and numeric scores. Boolean score values must equal either 1 or 0 (true or false)
       observationId: optional<string>
       comment: optional<string>
+      dataType:
+        type: optional<commons.ScoreDataType>
+        docs: When set, must match the score value's type. If not set, will be inferred from the score value or config
+      configId:
+        type: optional<string>
+        docs: Reference a score config on a score. When set, the score name must equal the config name and scores must comply with the config's range and data type. For categorical scores, the value must map to a config category. Numeric scores might be constrained by the score config's max and min values
+    docs: |
```

Comment:
I like the examples, should add on post /scores as well

Reply:
post /scores is more used via api than sdks, /ingestion only relevant for users building own tracing clients

Reply:
https://buildwithfern.com/learn/api-definition/fern/examples#objects